internal/pkg/cli/cmd/create.go

```diff
@@ -97,6 +99,15 @@ func CreateBranchCommand(root *RootCommand) *cobra.Command {
 				return err
 			}
 
+			// Send cmd successful/failed event
+			if eventSender, err := d.EventSender(); err == nil {
+				defer func() {
+					eventSender.SendCmdEvent(start, cmdErr, "create-branch")
```

Comment:
Vytvoreny novy event pre vytvorenie branch, kedze to sa deje priamo remote,  a ostatne operacie uz event maju.